spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -208,33 +208,113 @@ $ tail -f /var/folders/wn/8jxm_tbd1vj28c8vj37n900m0000gn/T/spring-cloud-dataflow
 2016-06-01 09:45:12.250  INFO 79194 --- [  kafka-binder-] log.sink    : 06/01/16 09:45:12
 2016-06-01 09:45:13.251  INFO 79194 --- [  kafka-binder-] log.sink    : 06/01/16 09:45:13
 ```
+=== Application properties
 
-=== Passing application properties when deploying stream
+Application properties are the properties associated with each application in the stream. When the application is deployed, the application properties are applied to the application via
+command line arguments or environment variables based on the underlying deployment implementation.
 
-When you execution `stream deploy` shell command, you can optionally provide a
-comma delimited list of key=value pairs known as `deployment properties`.
-These properties allow you to specify deployment instruction for each application
-in the stream.
+==== Passing application properties when creating stream
 
-==== Passing instance count as deployment property
+The following stream
 
-If you would like to have multiple instances of an application in the stream, you
-can include a property with the deploy command:
+[source,bash]
+----
+dataflow:> stream create --definition "time | log" --name ticktock
+----
+
+can have application properties defined at the time of stream creation.
+
+The shell command `app info` helps viewing the white-listed application properties for the application.
+For more info on the property white listing refer <<spring-cloud-dataflow-stream-app-whitelisting>>
+
+Below is the white listed properties for the app `time`:
 
-[source,bash,subs=attributes]
+[source,bash]
```

Comment:
shouldn't we show the `app info` command used here? (especially since the `id` format is not entirely intuitive)